packages/manager/apps/dedicated/client/app/billing/payment/method/routing.js

```diff
@@ -66,6 +80,73 @@ export default /* @ngInject */ (
           });
         }
       },
+      isSplitPaymentAvailable: /* @ngInject */ (
+        $http,
+        currentUser,
+        ovhFeatureFlipping,
+        splitPaymentTag,
+      ) =>
+        ovhFeatureFlipping
+          .checkFeatureAvailability(SPLIT_PAYMENT_FEATURE_NAME)
+          .then((feature) =>
+            feature.isFeatureAvailable(SPLIT_PAYMENT_FEATURE_NAME),
+          )
+          .then((isFeatureAvailable) =>
+            isFeatureAvailable
+              ? $http
+                  .get('/me/tag/available')
+                  .then(({ data: tags }) =>
+                    tags.some(
+                      ({ name: tagName }) => tagName === splitPaymentTag,
+                    ),
+                  )
```

Comment:
We need both feature flipping and checking tag availability ?

Reply:
Feature flipping is here to say that just IT customers can have the split payment. API may returns the tag regardless of the region